### PR TITLE
April Fools Day:  Oops, All Captains!

### PR DESCRIPTION
**Pull request by Skyedra**
## About the PR

(April Fool's day PR)

Adds a new game mode where _everyone_ is the captain.

Also, during the game mode, captains can be traitors to keep things interesting.  Role timers are also disabled during the game mode.

**Media**

Screenshot_20230328_231500

Screenshot_20230328_231634

Screenshot_20230328_231555

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: New game mode:  Oops, All Captains!

### CODE REVIEW

**Comment by DogZeroX** [comment]
This is literally just TTT! (SSS)

**Comment by Skyedra** [comment]
I don't know what your acronyms mean

**Comment by deltanedas** [comment]
trouble in captain town

**Comment by deltanedas** [comment]
also unless you are doing 2^31 - 1 is intentional theres an explicit thing for uncapped jobs, see passenger and the bureaucratic error that can make a job uncapped

File: Content.Server/GameTicking/Rules/AllCaptainsRuleSystem.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    public override void Ended()
+    {
+        _cfg.SetCVar(CCVars.GameRoleTimers, true);
+    }
+
+    public StationJobsComponent GetJobs(EntityUid station)
+    {
+        if (!holdJobs.IsValid() || !HasComp<StationJobsComponent>(holdJobs)) // this doesn't check station parameter since all captains mode is the same for all stations.
```
Comment (moonheart08) [inline Content.Server/GameTicking/Rules/AllCaptainsRuleSystem.cs:44]: how the fuck does this not explode violently

Comment (moonheart08) [reply Content.Server/GameTicking/Rules/AllCaptainsRuleSystem.cs:44]: @Skyedra oh
im sorry!
I'm just surprised. I can show you how to do this properly in a bit (currently a tad sick)
I did not mean that in any harsh way directed at you.

Comment (Skyedra) [reply Content.Server/GameTicking/Rules/AllCaptainsRuleSystem.cs:44]: No worries and thanks again for the detailed info over Discord, really helped me learn more about the codebase :+1:

**Pull request merged**